PubNubSwiftConsole/Classes/Components/ConsoleViewController.swift

```diff
@@ -210,7 +210,9 @@ public class ConsoleViewController: CollectionViewController, CollectionViewCont
                 return CGSize(width: 150.0, height: 100.0)
             case .ChannelPresenceButton, .ChannelGroupPresenceButton:
                 return CGSize(width: 200.0, height: 100.0)
-            case .SubscribeStatus, .Message, .All, .PublishStatus:
+            case .SubscribeStatus:
+                return CGSize(width: collectionViewSize.width, height: 230.0)
+            case .Message, .All, .PublishStatus:
```

Comment:
why different sizes for publish in the two commits? https://github.com/pubnub/SwiftConsole/pull/27/files#diff-b752841e26ee9f103fccbc3c17bb2662L41